File: src/main/java/com/example/misyemek/Service/FirmaSahibiService.java
package com.example.misyemek.Service;

import com.example.misyemek.entity.FirmaSahibi;
import com.example.misyemek.repository.FirmaSahibiRepository;
import org.springframework.stereotype.Service;

@Service
public class FirmaSahibiService {

    private final FirmaSahibiRepository repository;

    public FirmaSahibiService(FirmaSahibiRepository repository) {
        this.repository = repository;
    }

    public FirmaSahibi firmaSahibiGetir(Long kullaniciId){
        return repository.firmaSahibiGetir(kullaniciId);
    }
}

File: frontend/src/app/Service/firma-sahibi.service.ts
import { Injectable } from '@angular/core';
import { Observable } from 'rxjs';
import { BaseService } from '../core/services/base.service';

@Injectable({providedIn: 'root'})
export class FirmaSahibiService extends BaseService {

    protected baseUrl =`/FirmaSahibi`

    firmaSahibiGetir(): Observable<any>{
        return this.get<any>(`${this.kullaniciId}`)
    }
} 